app-backend/migrations/src/main/scala/migrations/51.scala

```diff
@@ -0,0 +1 @@
+../../../../src_migrations/main/scala/50.scala
```

Comment:
Oops, good catch! Yeah, I noticed the migration numbering conflict; will fix that up.